devops/github-pages.md

```diff
@@ -0,0 +1,16 @@
+# GitHub Pages Deployment
+How to host Good sites on GitHub pages.
+
+[![good](http://builtbygood.co/img/logo.png)](http://builtbygood.co)
+
+## Before site launch
+1. Create the repo in GitHub (private unless otherwise specified)
+2. When the site is ready for preview, create the `gh-pages` branch
+3. After pushing the gh-pages branch, the site will be live at builtbygood.co/`branch-name`
+
+
+## When it's time to launch
+1. Create a CNAME file per [this article](https://help.github.com/articles/setting-up-your-pages-site-repository/)
+2. High five
```

Comment:
Should we say that you need to push that file to the `gh-pages` branch as an extra explicit step?